### PR TITLE
Simplify example config

### PR DESCRIPTION
**Pull request by neekolas**
## tl;dr

- Simplifies the example to be a flat plan with no modules other than our public modules
- Renames the example
- Adds some outputs to existing modules
- Adds tflint

## TODO

- Still need to set up the DB

## Testing

I've tested this with `terraform validate` but haven't deployed it yet

### CODE REVIEW

**Comment by neekolas** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/xmtp/xmtpd-infrastructure/5?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5** <a href="https://app.graphite.dev/github/pr/xmtp/xmtpd-infrastructure/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/xmtp/xmtpd-infrastructure/5?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#4** <a href="https://app.graphite.dev/github/pr/xmtp/xmtpd-infrastructure/4?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3** <a href="https://app.graphite.dev/github/pr/xmtp/xmtpd-infrastructure/3?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2** <a href="https://app.graphite.dev/github/pr/xmtp/xmtpd-infrastructure/2?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: terraform/examples/aws-complete/aws [hunk 1]
```diff
@@ -0,0 +1 @@
+../../aws
```
Comment (richardhuaaa) [inline terraform/examples/aws-complete/aws:1]: In the base PR I actually switched to using GitHub paths with a branch, I actually forgot to point it back to main before landing:

`github.com/xmtp/xmtpd-infrastructure//terraform/aws/xmtp-validation-service?ref=rich-tf-validation-svc`

I wonder if that would be easier to work off of

**Review by richardhuaaa — APPROVED** [review]

File: terraform/examples/aws-complete/aws (hunk 1 above)
Comment (neekolas) [reply terraform/examples/aws-complete/aws:1]: That's the right place to leave the repo. It gets a little weird when you're trying to run `terraform validate` and you have any changes in the dependent modules.

You could push and then point at your current branch, but then you have to point back to main at some point...at which point the linter will fail. Almost need to break the PR into two pieces, one to update the module and one to update the dependency.

**Comment by neekolas** [comment]
### Merge activity

* **Nov 26, 4:22 PM PST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/xmtp/xmtpd-infrastructure/5).

File: terraform/examples/aws-complete/aws (hunk 1 above)
Comment (neekolas) [reply terraform/examples/aws-complete/aws:1]: I'll point it to `main` in the next PR, when `main` has stuff in the right place

**Pull request merged**